### DISCUSSION TITLE
archive extra files path

### DISCUSSION THREAD

**Discussion by taglme** [goreleaser/goreleaser#1570] [Ideas, Issue Triage, and General discussions]
Hi! In our project we want to add some extra files to archives. This files located in directory 
/assets/img/*
When we use "files" option in archives section like that
  files:
    - assets/img/*
 goreleaser adds this files with entire path to the archive root
 assets/img/*
Is it possible to somehow strip path and add to the archive root folder
img/*
or add files directly to archive root?

**Comment by jportasa** [comment]
I am as well interested in it.

**Comment by caarlos0** [comment]
right now its not possible, no...

**Reply by ainsleyclark** [reply]
@caarlos0 is this something you will be looking to implement at any point?

**Reply by caarlos0** [reply]
yes, there is an issue for it https://github.com/goreleaser/goreleaser/issues/1638
